account-management-app/src/main/resources/root.properties

```diff
@@ -68,6 +68,8 @@ notifications.emailtemplates.USER_ADDED_TO_ACCOUNT.description=The email sent to
 storageprovider.storagelimit=Storage Limit (in TB)
 storageprovider.writablebynonrootuser=Writable by non-root users
 amazon.region=Amazon Region (default empty)
+s3.endpoint=S3 Endpoint
+s3.signer.type=Signer Type
```

Comment:
I suggested on the DuraCloud PR (https://github.com/duracloud/duracloud/pull/92) that these be named so that it's clear they are relevant to the Swift implementation. Whatever changes are made there will require changes here for consistency.